motor/providers/awsec2ebs/types.go

```diff
@@ -53,18 +53,20 @@ type SnapshotId struct {
 	Account string
 }
 
-type VolumeId struct {
+type VolumeInfo struct {
 	Id          string
 	Region      string
 	Account     string
 	IsAvailable bool
+	Tags        map[string]string
 }
 
 func resourceTags(resourceType types.ResourceType, instanceId string) []types.TagSpecification {
 	return []types.TagSpecification{
-		{ResourceType: resourceType,
+		{
+			ResourceType: resourceType,
 			Tags: []types.Tag{
-				{Key: aws.String("Created By"), Value: aws.String("Mondoo")},
+				{Key: aws.String("createdBy"), Value: aws.String("Mondoo")},
```

Comment:
is there a reason this was changed from "Created By" to "createdBy" ? i believe we have other logic that depends on this tag naming in the lambda

Reply:
Was requested here: https://github.com/mondoohq/cnquery/pull/504#discussion_r1031076186

Because of compliance with the standard AWS tags.

Where can we find out whether we need this specific form of the tag?

Reply:
https://gitlab.com/mondoolabs/mondoo/-/blob/master/apps/mondoo-lambda/cloudformation/mondoo-lambda-cf.json#L490 we currently depend on the tag being "Created By": "Mondoo"

Reply:
can we add both tags for now, and then we'll transition the lambda to the new one? i'll make a ticket for the followup

Reply:
Sure. Added it.